projects/decoder_tests/src/lib.rs

```diff
@@ -326,15 +343,217 @@ fn test_issue() {
     let _ = trace_data("  IOU Asset:", &buf[..output_len as usize], DataRepr::AsHex);
 }
 
+fn test_deposit_preauth() {
```

Comment:
Can these tests be alphabetized? Easier to see if something is missing that way

Reply:
done